README.md

```diff
@@ -209,6 +209,19 @@ TrailingWhitespace
 UnorderedKey
 ```
 
+`dotenv-linter` can also automatically fix warnings in the files. Currently only one kind of warnings is fixed
+(`LowercaseKey`). You should use the argument `--fix` (or its short version `-f`) for this.
+
+```shell script
+$ dotenv-linter -f
+Fixed warnings:
+.env:3 LowercaseKey: The foo key should be in uppercase
+
+Unfixed warnings:
+.env:2 DuplicatedKey: The BAR key is duplicated
+  
```

Comment:
Extra blank line detected 😉

Reply:
There is a blank line after a warning group in your example :slightly_smiling_face::

> Fixed warnings:
> subdir/.env:2 LowercaseKey: The foo key should be in uppercase
> subdir/.env.common:3 LowercaseKey: The foo key should be in uppercase
> subdir/test.env:3 LowercaseKey: The var key should be in uppercase
> 
> Unfixed warnings:
> .env:1 KeyWithoutValue: The FOO key should be with a value or have an equal sign
> .env:2 LeadingCharacter: Invalid leading character detected
> .env:2 KeyWithoutValue: The  BAR key should be with a value or have an equal sign

Should I remove it?

Reply:
Or leave a blank line between groups, but remove it at the end?

Reply:
Please, remove extra blank line at the end.

Reply:
Done